source/logHandler.py

```diff
@@ -266,8 +266,13 @@ def emit(self, record):
 class FileHandler(logging.FileHandler):
 
 	def handle(self,record):
-		# Only play the error sound if this is a test version.
-		shouldPlayErrorSound =  buildVersion.isTestVersion
+		import config
+		# Only play the error sound if this is a test version or if the config states it explicitely.

```

Comment:
explicitely -> explicitly
```suggestion
		# Only play the error sound if this is a test version or if the config states it explicitly.
```